src/server/graphql/mutations/downgradeToPersonal.ts

```diff
@@ -66,18 +68,18 @@ export default {
         )('changes')('new_val')('id')
         .default([])
     })
-    sendSegmentEvent('Downgrade to personal', userId, {orgId})
+    sendSegmentEvent('Downgrade to personal', viewerId, {orgId})
     const data = {orgId, teamIds}
     publish(ORGANIZATION, orgId, DowngradeToPersonalPayload, data, subOptions)
 
     teamIds.forEach((teamId) => {
       // I can't readily think of a clever way to use the data obj and filter in the resolver so I'll reduce here.
-      // This is probably a smelly piece of code telling me I should be sending this per-userId or per-org
+      // This is probably a smelly piece of code telling me I should be sending this per-viewerId or per-org
       const teamData = {orgId, teamIds: [teamId]}
       publish(TEAM, teamId, DowngradeToPersonalPayload, teamData, subOptions)
     })
     // the count of this users tier stats just changed, update:
-    await sendSegmentIdentify(userId)
+    await sendSegmentIdentify(viewerId)
```

Comment:
I don't think this should be `viewerId`. In fact, I failed you on the first review: this should be an array of all affected userIds (all of the userIds on the org just downgraded). The resulting identify events will propagate the new tier counts to all of our systems (HubSpot, Mixpanel, etc.)

Reply:
good catch